doc/sphinx-guides/source/installation/config.rst

```diff
@@ -324,10 +392,25 @@ Optionally, you can have users download files from S3 directly rather than havin
 
 If you enable ``dataverse.files.s3-download-redirect`` as described above, note that the S3 URLs expire after an hour by default but you can configure the expiration time using the ``dataverse.files.s3-url-expiration-minutes`` JVM option. Here's an example of setting the expiration time to 120 minutes:
 
-``./asadmin create-jvm-options "-D dataverse.files.s3-url-expiration-minutes=120"``
+``./asadmin create-jvm-options "-Ddataverse.files.s3-url-expiration-minutes=120"``
 
 Lastly, go ahead and restart your glassfish server. With Dataverse deployed and the site online, you should be able to upload datasets and data files and see the corresponding files in your S3 bucket. Within a bucket, the folder structure emulates that found in local file storage.
 
+S3 Storage Options
+##################
+
+=========================================  ==================  ==================================================================  =============
+System Property                            Value               Description                                                         Default value
+=========================================  ==================  ==================================================================  =============
+dataverse.files.storage-driver-id          s3                  Enable S3 storage driver.                                           ``file``
+dataverse.files.s3-bucket-name             <?>                 The bucket name. See above.                                         (none)
+dataverse.files.s3-download-redirect       ``true``/``false``  Enable direct download or proxy through Dataverse.                  ``false``
+dataverse.files.s3-url-expiration-minutes  <?>                 If direct downloads: time until links expire. Optional.             60
+dataverse.files.s3-url                     <?>                 Use custom S3 endpoint. Needs URL either with or without protocol.  (none)
+dataverse.files.s3-region                  <?>                 Only used when using custom endpoint. Optional.                     ``dataverse``
```

Comment:
Credit to @matthew-a-dunlap for pointing this out but I'm wondering if "s3-region" is a little misleading since it has no effect on S3. I know the description explains it's only for the custom (non-S3) endpoint but should we use a different name? I'm not sure what other name would even make sense since all these options start with "s3".

Reply:
Actually I thought about renaming the options altogether, as they seem to have grown to a number where a separate namespace/scope seems to make sense (think of sth. like `dataverse.files.s3.xxx`. Of course this would break existing setups :-1: 

About the renaming: unsure. It definitly is the region. Setting it on installations without a custom URL will do simply nothing.

Maybe we could rename both options to `dataverse.files.s3-custom-endpoint-url` and `dataverse.files.s3-custom-endpoint-region`? But anyway: I'm unsure if there is much benefit in that.

Reply:
Actually, I like those names better. Good idea. These:

- dataverse.files.s3-custom-endpoint-url
- dataverse.files.s3-custom-endpoint-region

More descriptive and helps differentiate them from the true S3.

Reply:
Will do so... :smile: